src/e2e-tests/features/autelis.feature

```diff
@@ -0,0 +1,96 @@
+@autelis
+Feature: RoboDomo Autelis page
+
+
+    @S1 @automated
+    Scenario Outline: Check Autelis pool/spa/solar options
+        Given User loads the RoboDomo web app
+        When User clicks Autelis menu button
+        Then User is redirected to Autelis page
+        Then Autelis tab shall be selected
+        When User clicks on Autelis tab
+        When User clicks on <offButton> button on Autelis page
+        Then All Autelis SPA-POOL parameters should be turned off
+        When User clicks on <poolButton> button on Autelis page
+        Then Pool parameters are displayed
+        When User clicks on <spaButton> button on Autelis page
+        Then Spa parameters are displayed
+        And Solar parameters are displayed
+        Examples:
```

Comment:
if you want this test to be executed only once then `Outline` is not needed. The problem here is that if one fails the rest are not tested

If you want to test these options independently then here is a solution:
```
Scenario Outline: Check Autelis pool/spa/solar options
        Given User loads the RoboDomo web app
        When User clicks Autelis menu button
        Then User is redirected to Autelis page
        Then Autelis tab shall be selected
        When User clicks on Autelis tab
        When User clicks on <option> button on Autelis page
        Then Autelis <parameters> parameters should be turned off
        Examples:
            | option  | parameters |
            | OFF      | SPA-POOL|
            |POOL    | POOL |
            |SPA        | SPA-SOLAR |
```

Also, if you choose this approach then please rework the step definitions also.
Thank you

Reply:
Updated